jetty-server/src/main/java/org/eclipse/jetty/server/HttpInput.java

```diff
@@ -156,7 +156,7 @@ public int read(byte[] b, int off, int len) throws IOException
                 long period=System.nanoTime()-_firstByteTimeStamp;
                 if (period>=1000)
                 {
-                    double data_rate = _contentArrived / ((1.0*period)/TimeUnit.SECONDS.toNanos(1));
+                    double data_rate = _contentArrived / (((double)period)/TimeUnit.SECONDS.toNanos(1));
```

Comment:
Why you want to do this I don't understand.

You add magic numbers into the code (`1000`, which is wrong by the way - should be `1000000`), unnecessary conversion to `double`, `double` to `long` comparisons (which also require conversions) when I already posted much simpler code that just uses `long` and has no magic numbers. 

Please, just use my version.